src/qibocal/protocols/characterization/fast_reset/fast_reset.py

```diff
@@ -0,0 +1,239 @@
+
+
+def _plot(data: FastResetData, fit: FastResetResults, qubit):
+    """Plotting function for FastReset."""
+
+    # Maybe the plot can just be something like a confusion matrix between 0s and 1s ???
+
+    figures = []
+    fitting_report = ""
+    fig = make_subplots(
+        rows=1,
+        cols=2,
+        horizontal_spacing=0.1,
+        vertical_spacing=0.1,
+        subplot_titles=(
+            "Fast Reset",
+            "Relaxation Time",
+        ),
+    )
+
+    # state 1
+    fr_states = data[qubit, 1, True].probability
+    nfr_states = data[qubit, 1, False].probability
+
+    nshots = len(fr_states)
+
+    # FIXME crashes if all states are on the same counts
+    unique, counts = np.unique(fr_states, return_counts=True)
+    state0_count_1fr = counts[0]
+    state1_count_1fr = counts[1]
+    error_fr1 = 1 - (nshots - state0_count_1fr) / nshots
+
+    unique, counts = np.unique(nfr_states, return_counts=True)
+    state0_count_1nfr = counts[0]
+    state1_count_1nfr = counts[1]
+    error_nfr1 = 1 - (nshots - state0_count_1nfr) / nshots
+
+    # state 0
+    fr_states = data[qubit, 0, True].probability
+    nfr_states = data[qubit, 0, False].probability
+
+    unique, counts = np.unique(fr_states, return_counts=True)
+    state0_count_0fr = counts[0]
+    state1_count_0fr = counts[1]
+    error_fr0 = (nshots - state0_count_0fr) / nshots
+
+    unique, counts = np.unique(nfr_states, return_counts=True)
+    state0_count_0nfr = counts[0]
+    state1_count_0nfr = counts[1]
+    error_nfr0 = (nshots - state0_count_0nfr) / nshots
+
+    fig.add_trace(
+        go.Heatmap(
+            z=[
+                [state0_count_0fr, state0_count_1fr],
+                [state1_count_0fr, state1_count_1fr],
+            ],
+            colorbar_x=0.45,
+        ),
+        row=1,
+        col=1,
+    )
+
+    fig.update_xaxes(
+        title_text=f"{qubit}: Fast Reset",
+        row=1,
+        col=1,
+    )
+    fig.update_yaxes(title_text="State", row=1, col=1)
+    fig.update_yaxes(tickvals=[0, 1])
+    fig.update_xaxes(tickvals=[0, 1])
+
+    fig.add_trace(
+        go.Heatmap(
+            z=[
+                [state0_count_0nfr, state0_count_1nfr],
+                [state1_count_0nfr, state1_count_1nfr],
+            ],
+        ),
+        row=1,
+        col=2,
+    )
+
+    fig.update_layout(coloraxis={"colorscale": "viridis"})
+
+    fig.update_xaxes(
+        title_text="State prepared",
+        row=1,
+        col=2,
+    )
+
+    fitting_report += f"q{qubit}/r | Error FR0: {error_fr0:.6f}<br>"
+    fitting_report += f"q{qubit}/r | Error FR1: {error_fr1:.6f}<br>"
+    fitting_report += (
+        f"q{qubit}/r | Fidelity FR: {(1 - (error_fr0 + error_fr1)/2):.6f}<br>"
+    )
+    fitting_report += f"q{qubit}/r | Error NFR0: {error_nfr0:.6f}<br>"
+    fitting_report += f"q{qubit}/r | Error NFR1: {error_nfr1:.6f}<br>"
+    fitting_report += (
+        f"q{qubit}/r | Fidelity NFR: {(1 - (error_nfr0 + error_nfr1)/2):.6f}<br>"
+    )
```

Comment:
```suggestion
    fitting_report += f"{qubit} | Error FR0: {error_fr0:.6f}<br>"
    fitting_report += f"{qubit} | Error FR1: {error_fr1:.6f}<br>"
    fitting_report += (
        f"{qubit} | Fidelity FR: {(1 - (error_fr0 + error_fr1)/2):.6f}<br>"
    )
    fitting_report += f"{qubit} | Error NFR0: {error_nfr0:.6f}<br>"
    fitting_report += f"{qubit} | Error NFR1: {error_nfr1:.6f}<br>"
    fitting_report += (
        f"{qubit} | Fidelity NFR: {(1 - (error_nfr0 + error_nfr1)/2):.6f}<br>"
    )
```
Here it would be better to write more explicitly in the table what are the errors.